bin/travis

```diff
@@ -44,28 +42,14 @@ if [[ $TRAVIS_BRANCH = "master" || $TRAVIS_BRANCH = "release-"* || $TRAVIS_BRANC
   fi
 fi
 
-
-# Put JVM options in a file where we can write to it without sudo,
-# for instance to adjust the JVM's entropy source.
-JVM_OPTS_FILE=/tmp/jvmopts
-cp /etc/sbt/jvmopts "$JVM_OPTS_FILE"
-
+# Using /dev/urandom in Travis CI to avoid hanging on /dev/random
+# when VM or container entropy entropy is low.  Additional detail at
+# https://github.com/http4s/http4s/issues/774#issuecomment-273981456
 if [[ "$TRAVIS" = "true" ]] && [[ -r "/dev/urandom" ]]; then
-    echo
-    echo "Using /dev/urandom in Travis CI to avoid hanging on /dev/random"
-    echo "when VM or container entropy entropy is low.  Additional detail at"
-    echo "https://github.com/http4s/http4s/issues/774#issuecomment-273981456 ."
-
-    echo "-Djava.security.egd=file:/dev/./urandom" >> "$JVM_OPTS_FILE"
-    echo
-    echo "JVM_OPTS:"
-    for opt in $(grep -v -e '^#' -e '^$' "$JVM_OPTS_FILE"); do
-        echo "  $opt"
-    done
-    echo
+  export SBT_OPTS="$SBT_OPTS -Djava.security.egd=file:/dev/./urandom"
```

Comment:
You can probably just remove the `-Djava.security.egd=file:/dev/./urandom"` bit.  I added that thinking we might have been blocking on lack of entropy, but I really doubt that's the case after spending more time with the build.  Too little entropy used to be a problem on some of the Travis infrastructure, but I think that's been fixed for a long time now.